<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QTextEdit" name="txtGrammar"/>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnErrors">
          <property name="text">
           <string>Ошибки</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/error.png</normaloff>:/icons/icons/error.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWarnings">
          <property name="text">
           <string>Предупреждения</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/warning.png</normaloff>:/icons/icons/warning.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="listMessages">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mnuCreate"/>
    <addaction name="mnuOpen"/>
    <addaction name="mnuSave"/>
    <addaction name="mnuSaveAs"/>
    <addaction name="separator"/>
    <addaction name="mnuPrint"/>
    <addaction name="separator"/>
    <addaction name="mnuExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="mnuUndo"/>
    <addaction name="mnuRedo"/>
    <addaction name="separator"/>
    <addaction name="mnuCut"/>
    <addaction name="mnuCopy"/>
    <addaction name="mnuPaste"/>
    <addaction name="mnuDelete"/>
    <addaction name="separator"/>
    <addaction name="mnuFind"/>
    <addaction name="mnuFindNext"/>
    <addaction name="mnuReplace"/>
    <addaction name="mnuGoto"/>
    <addaction name="separator"/>
    <addaction name="mnuSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuAjhvfn">
    <property name="title">
     <string>Формат</string>
    </property>
    <addaction name="mnuWordWrap"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="mnuHelp"/>
    <addaction name="separator"/>
    <addaction name="mnuAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuAjhvfn"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mnuCreate"/>
   <addaction name="mnuSave"/>
   <addaction name="separator"/>
   <addaction name="mnuRecursionCheck"/>
   <addaction name="mnuBuildMatrixSimple"/>
   <addaction name="separator"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnuCreate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/newfile.png</normaloff>:/icons/icons/newfile.png</iconset>
   </property>
   <property name="text">
    <string>Новая грамматика</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="mnuOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mnuSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mnuSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="mnuPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/print.png</normaloff>:/icons/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Печать...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="mnuExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="mnuUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="mnuRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/redo.png</normaloff>:/icons/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Повторить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="mnuCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/cut.png</normaloff>:/icons/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="mnuCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/copy.png</normaloff>:/icons/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="mnuPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/paste.png</normaloff>:/icons/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="mnuDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mnuFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/find.png</normaloff>:/icons/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Найти...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="mnuFindNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/find.png</normaloff>:/icons/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Найти далее</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="mnuReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/replace.png</normaloff>:/icons/icons/replace.png</iconset>
   </property>
   <property name="text">
    <string>Заменить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="mnuGoto">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/goto.png</normaloff>:/icons/icons/goto.png</iconset>
   </property>
   <property name="text">
    <string>Перейти...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="mnuSelectAll">
   <property name="text">
    <string>Выделить все</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="mnuHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Просмотреть справку</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="mnuAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/about.png</normaloff>:/icons/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="mnuWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перенос по словам</string>
   </property>
  </action>
  <action name="mnuOptions">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="mnuRecursionCheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/check.png</normaloff>:/icons/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Проверка рекурсивности правил</string>
   </property>
  </action>
  <action name="mnuBuildMatrixSimple">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/matrix-simple.png</normaloff>:/icons/icons/matrix-simple.png</iconset>
   </property>
   <property name="text">
    <string>Построить матрицу простого предшествования</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnErrors</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefreshMessagesList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWarnings</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefreshMessagesList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRefreshMessagesList()</slot>
 </slots>
</ui>
